pages/features.html

```diff
@@ -6,92 +6,122 @@
 
 {% block body %}
 <div class="content">
-  <H1>{% trans "World-Class Mixing Engine" %}</H1>
-  {% trans "Don't just beatmix, beatmix better." %}
+  <h1>{% trans "Precise Mixing Engine" %}</h1>
+  {% trans "Essential tools for seemlessly blending music" %}
 
   <div style="clear: both;"></div>
   <div class="halfbox_left">
-    <H2>{% trans "Four Decks, Advanced Controls" %}</H2>
+    <h2>{% trans "Four Decks" %}</h2>
     <p>
-      {% trans "Drop a song onto a deck in Mixxx to kick off your next mix. Each deck features a scratchable, scrolling waveform that marks beats and cue points, along with a whole-song waveform overview for quick seeking." %}
+      {% trans "Drop a song onto a deck in Mixxx to start playing right away. Each deck features a scratchable, scrolling waveform that marks beats and cue points, along with a whole-song waveform overview for quick seeking." %}
     </p>
 
-    <p><b>{% trans "Time Stretch and Vinyl Emulation" %}</b><br>
-      {% trans "Speed up and slow down songs without changing their pitch with time stretching, or just change the musical pitch so your songs play in harmony. Our high-quality interpolator can also reproduce classic vinyl sounds like backspins." %}
+    <p><b>{% trans "Pitch and Key Control" %}</b><br>
+      {% trans "Change the tempo of songs without changing their pitch with keylock, or change the musical pitch apart from tempo so your songs play in harmony. Ramping pitchbend ensures any tempo nudging is smoothly applied, just like with vinyl." %}
     </p>
 
     <p><b>{% trans "Beat Looping" %}</b><br>
-      {% trans "Found a sweet loop or need to extend your mix while you prep the next track? Instantly loop a 4, 8, or 16 beat segment with the click of a button." %}
+      {% trans "Need to extend your mix while you bring in the next track? Loop a 4, 8, or 16 beat segment with ease." %}
     </p>
 
-    <p><b>{% trans "Powerful Master Sync" %}</b><br>
+    <p><b>{% trans "Master Sync" %}</b><br>
       {% trans "Engage master sync on your decks and they'll stay locked in time even if you change speed.  Construct remixes on the fly with multiple tracks and loops without losing control." %}
     </p>
 
     <p><b>{% trans "Hotcues" %}</b><br>
-      {% trans "Four hotcue points can be dropped or triggered with ease. Our playback engine is finely tuned for accurate, rapid fire hotcue triggering so you can mash and remix as hard as you want to without a hiccup." %}
+      {% trans "Set and trigger up to four hotcues. Our playback engine is finely tuned for accurate, rapid fire hotcue triggering so you can mash and remix as hard as you want to without a hiccup." %}
```

Comment:
The "four" is a skin only limit. Here we have again command style mixed with explanations.